### PR TITLE
[GCI-142] Removed class fields shadowing

### PR DESCRIPTION
**Pull request by tomekdo3**
### Description of what I changed
Removed class fields shadowing

### Issue I worked on

https://issues.openmrs.org/browse/GCI-142

### CODE REVIEW

File: api/src/main/java/org/openmrs/Concept.java [hunk 1]
```diff
@@ -1600,8 +1600,8 @@ public void addSetMember(Concept setMember, int index) {
 		//sets have moved to the bottom and hence need to be reassigned
 		//higher sort weights than the non retired ones
 		double weight = 990.0;
-		for (ConceptSet set : sortedConceptSets) {
-			set.setSortWeight(weight += 10.0);
+		for (ConceptSet conceptSet : sortedConceptSets) {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/Concept.java]: Do we have a class field named conceptSet?

Comment (tomekdo3) [reply api/src/main/java/org/openmrs/Concept.java]: No we don't have it as class field, only as local

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/Concept.java]: Do we have a class field named set?

Comment (tomekdo3) [reply api/src/main/java/org/openmrs/Concept.java]: We do have one. It's a Boolean

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/Concept.java]: Can you point me to it with a github link?

Comment (tomekdo3) [reply api/src/main/java/org/openmrs/Concept.java]: https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Concept.java#LN98
line 98

**Comment by dkayiwa** [comment]
Thanks @tomekdo3 😊

**Pull request merged**